hudi-common/src/test/java/org/apache/hudi/common/util/collection/TestExternalSpillableMap.java

```diff
@@ -339,7 +343,41 @@ public void testDataCorrectnessWithoutHoodieMetadata(ExternalSpillableMap.DiskMa
 
   // TODO : come up with a performance eval test for spillableMap
   @Test
-  public void testLargeInsertUpsert() {}
+  public void testLargeInsertUpsert() {
+  }
+
+  @ParameterizedTest
+  @MethodSource("testArguments")
+  public void testPayloadSizeEstimate(ExternalSpillableMap.DiskMapType diskMapType,
```

Comment:
I guess we don't need to do parametrized here.

Reply:
right, the issue is in the Map wrapper class and not in the backing map. Removed the parameterization.